viewer/pom.xml

```diff
@@ -11,9 +11,9 @@
 
     <dependencies>
         <dependency>
-            <groupId>uk.co.real-logic</groupId>
-            <artifactId>Agrona</artifactId>
-            <version>0.3.1</version>
+            <groupId>org.kaazing</groupId>
+            <artifactId>metrics.reader</artifactId>
+            <version>develop-SNAPSHOT</version>
```

Comment:
We haven't decided yet on how the versioning system for this project should be. This will be decided in a future sprint, therefore that's why we kept the "develop-SNAPSHOT" for the moment.